packages/python/plotly/setup.py

```diff
@@ -505,7 +505,7 @@ def run(self):
         ),
         ("etc/jupyter/nbconfig/notebook.d", ["plotlywidget.json"]),
     ],
-    install_requires=["retrying>=1.3.3", "six"],
+    install_requires=["retrying>=1.3.3", "six", 'pathlib2;python_version<"3.4"'],
```

Comment:
Let's remove this, as we're dropping support for 2.7 and 3.5 in the next Plotly.py release.